modules/installation-azure-user-infra-wait-for-bootstrap.adoc

```diff
@@ -27,12 +27,6 @@ the Ignition config files that you generated with the installation program.
 
 .Prerequisites
 
-* Configure an Azure account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VNet and associated subnets in {cp}.
-* Create and configure networking and load balancers in {cp}.
-* Create control plane and compute roles.
-* Create the bootstrap machine.
 * Create the control plane machines.
```

Comment:
Can the whole thing be removed?

Reply:
This is the first instance of ' Create the control plane machines.' in the assembly, so I think it needs to stay.